windows-active-directory-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,79 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'windows-active-directory-alerts',
+        rules: [
+          {
+            alert: 'WindowsActiveDirectoryHighPendingReplicationOperations',
+            expr: |||
+              windows_ad_replication_pending_operations >= %(alertsHighPendingReplicationOperations)s 
+            ||| % $._config,
+            'for': '10m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'There is a high number of pending replication operations in Active Directory. A high number of pending operations sustained over a period of time can indicate a problem with replication.',
```

Comment:
I feel the second sentence being here defeats the purpose of this summary section. As far as I know and how I've treated it, this is just a quick blurb about the alert, so I feel the first sentence is enough for the summary. I think the second sentence would fit better in the description. Ultimately your call though if you've gotten review in the past that says otherwise. 

This comment extends to HighReplicationSyncRequestFailures and HighPasswordChanges

Reply:
Im not sure about this, I have always seen the description as being a flat out written form of where the thresholds have been crossed at what instances. The summary to me has always been why whatever alert has been triggered is an actual problem. I think I will keep it as is for now.

Reply:
That's fine, just something to keep in mind cus in the UI the summary doesn't have a lot of space, if you check out some of the pre-existing alerts on the cloud instance